templates/pro.html

```diff
@@ -0,0 +1,566 @@
+            <p>Ubuntu Proは、オンプレミスやパブリッククラウド（AWS、Azure、GCPなど）でコンプライアンスを維持するツールを提供します。</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="u-no-margin--bottom is-muted" />
+      <div class="col">
+        <h3 class="p-heading--2">Ubuntu Proのアドオン</h3>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-large-4">
+        <div class="row">
+          <hr class="is-muted"/>
+          <div class="col-3">
+            <strong>24時間体制の電話とチケットによるサポート</strong>
+          </div>        
+          <div class="col-9">
+            <div class="p-image-container--16-9 is-highlighted">
+              {{ image(url="https://assets.ubuntu.com/v1/50dea393-fixes+in+24hrs.svg",
+                        alt="",
+                        width="145",
+                        height="150",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+              }}
+            </div>
+            <p>環境に不具合が生じたときは電話とチケットによるサポートをトラブルシューティングやバグ修正にご活用ください。</p>
+            <p>Ubuntu Pro +サポートは、インフラストラクチャレイヤー（OpenStack、MAAS、Ceph、LXD、Kubernetes）から、ベースとなるUbuntu OS、Ubuntuリポジトリ内のすべてのアプリケーションパッケージと依存ファイル（Kafka、Kubeflow、OpenJDK、PostgreSQLなど）まで、スタック全体に対応します。</p>
+            <hr class="is-muted" />
+            <p>
+              <a href="/contact-us" class="p-button">お問い合わせ</a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-large-4">
+        <div class="row">
+          <hr class="is-muted"/>
+          <div class="col-3">
+            <strong>マネージドサービスへのアップグレード</strong>
+          </div>        
+          <div class="col-9">
+            <div class="p-image-container--16-9 is-highlighted">
+              {{ image(url="https://assets.ubuntu.com/v1/91b86713-Enterprise_support.svg",
+                        alt="",
+                        width="130",
+                        height="150",
+                        hi_def=True,
+                        loading="auto|lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+              }}
+            </div>
+            <p>重大な問題が発生したときに緊急サポートでCanonicalのエンジニアのアドバイスを受けることも、フルマネージドソリューションで環境のセットアップから実運用までCanonicalに任せることも可能です。</p>
+            <p>マネージドサービスは、インフラストラクチャレイヤー全体（MAAS、Ceph、OpenStack、MicroCloud、LXD、Kubernetes）のほか、多数のアプリケーション（Kafka、Kubeflow、Grafana、PostgreSQLなど）を含みます。</p>
+            <hr class="is-muted" />
+            <p>
+              <a href="/engage/hybrid-cloud-journey" class="p-button">参考資料</a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <hr class="u-fixed-width is-muted"/>
+  <section class="p-strip is-deep">
+    <div class="row">
+      <h2>詳しい情報を入手する</h2>
+    </div>
+    <div class="row">
+      <div class="col">
```

Comment:
`col` is only valid when you use a row with predefined cols (eg row--50-50), but this whole section can live in that `u-fixed-width` honestly, no need for the extra row / col